imagestreams/quarkus-centos7.json

```diff
@@ -0,0 +1,54 @@
+{
+    "apiVersion": "v1",
+    "kind": "ImageStream",
+    "metadata": {
+        "annotations": {
+            "openshift.io/display-name": "Quarkus"
+        },
+        "name": "quarkus"
+    },
+    "spec": {
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Build and run Java (Quarkus) applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/quarkus/quarkus-images.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Quarkus available on OpenShift, including major versions updates.",
```

Comment:
That's actually not true. It will use the grallVM version associated with the image. The quarkus version is still managed by the user.